src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -67,6 +69,8 @@ export class CloudsyncFormComponent implements OnInit {
       : this.translate.instant('Edit Cloud Sync Task');
   }
 
+  CloudCredentialsFormComponent = CloudCredentialsFormComponent;
```

Comment:
That's weird. Maybe we can follow other approaches. For example, what if we can do a map of possible options and components and then use an enum to avoid importing the whole component like that?